classes/plugin/__init__.py

```diff
@@ -0,0 +1,48 @@
+from . import api
```

Comment:
dont add this code in `__init__.py`. put this into another file called `plugins.py` or something. only use `__init__.py` to make loading modules easier.